api/src/main/java/org/openmrs/module/eptsreports/service/EptsGlobalPropertyServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.openmrs.module.eptsreports.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.impl.BaseOpenmrsService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EptsGlobalPropertyServiceImpl extends BaseOpenmrsService
+    implements EptsGlobalPropertyService {
+
+  private static Log log = LogFactory.getLog(EptsGlobalPropertyServiceImpl.class);
+
+  @Override
+  public List<GlobalProperty> removeEptsGlobalPropertiesEntries(String patternName) {
+
+    List<GlobalProperty> remvedProperties = new ArrayList<>();
+
+    AdministrationService administrationService = Context.getAdministrationService();
+    List<GlobalProperty> globalProperties = administrationService.getAllGlobalProperties();
```

Comment:
Retrieving by prefix could save us the `if` conditional bellow.
```suggestion
    List<GlobalProperty> globalProperties = administrationService.getGlobalPropertiesByPrefix(patternName);
```